package.json

```diff
@@ -72,9 +72,12 @@
   "devDependencies": {
     "@commitlint/cli": "^11.0.0",
     "@commitlint/config-conventional": "^11.0.0",
+    "all-contributors-cli": "^6.19.0",
```

Comment:
probably don't need to include it.

Reply:
I was on the fence about this.  I figure if we're gonna use it, let's be transparent about what we're using.  Plus, it's just a dev dependency.  Up for discussion, but as I did this, I thought 'why not?'

Reply:
removed